docs/developer/payments/lifecycle.mdx

```diff
@@ -78,6 +78,34 @@ Charge status depends on:
 
 The difference between `total` and `totalGrantedRefund` is compared with the sum of **charged** and **authorized** transactions. The result is one of the statuses: `PARTIAL`, `FULL`. If no transaction has been charged, the status is `NONE`.
 
+#### Authorization status calculation for Checkout
+
+The status is calculated based on all transactions assigned to the `Checkout`. 
+
+For a checkout, authorization means that funds have been either authorized or charged (or a combination of partially charged and partially authorized).
+The status takes into account the following fields: `authorizedAmount`, `chargedAmount`, `authorizePendingAmount`, and `chargePendingAmount`:
+
+- **NONE**: No funds have been authorized.
+- **PARTIAL**: The available funds do not fully cover the checkout's total amount.
+- **FULL**: The funds fully cover the checkout's total amount. At this point, `checkoutComplete` can be called to convert the checkout into an order. 
+
+Note that pending amounts (success or failure) will be confirmed later by the payment app. This means the system can create an order that may remain unpaid if the pending amounts eventually fail.
+
+#### Available transitions
+
+- Status is updated on operations that change the checkout's `total`, or transactions.
+
+#### Authorization status calculation for Order
+
+The status is calculated based on all transactions assigned to the `Order`.
+Authorization means that funds have been either fully or partially authorized or charged.
+Unlike checkout, this status does not consider pending amounts.
```

Comment:
I think this should be put again into caution / warning callout, since this is not very intuitive 😅